tests/components/uptimerobot/common.py

```diff
@@ -76,7 +86,9 @@ def mock_uptimerobot_api_response(
     )
 
 
-async def setup_uptimerobot_integration(hass: HomeAssistant) -> MockConfigEntry:
```

Comment:
Revert the changes here, we do not want a separate function for each platform.
You can assert the sensor inside this comment function